avm/src/vm/memory.rs

```diff
@@ -380,9 +394,9 @@ impl HandlerMemory {
       }
       // copy fractal from ancestor
       let addr = fractal.hm_addr.as_ref().unwrap().clone();
-      let (_, hm_opt) = self.addr_to_idxs(addr);
+      let ((a, _), hm_opt) = self.addr_to_idxs(addr);
       let hm = hm_opt.expect("couldn't write fractal to HandlerMemory: address doesn't exist");
-      HandlerMemory::transfer(&hm, addr, self, addr);
+      Arc::get_mut(self).expect("how").mems[a] = hm.mems[a].clone();
       drop(hm);
       todo!()
```

Comment:
forgot about this `todo!()` - since we haven't hit this path but we still need to cover it I think it's fair to remove it? I inserted it when I was changing everything to use `Arc<HandlerMemory>` since I thought I'd have to do more here...

Reply:
So drop the entire branch, or just the `todo!()` call?

Reply:
just the `todo!()` call - although we can keep it and add a message asking the user to report to us their use case so we can reproduce and it's correct, which would make debugging their probably-wrong program easier?

Reply:
Let's go with the latter; I don't think our test suite is comprehensive enough to even consider an end user's program "probably wrong." More likely "AVM is probably wrong", I think?